platform/mbed_retarget.h

```diff
@@ -559,7 +561,9 @@ struct pollfd {
 #if __cplusplus
 extern "C" {
 #endif
+#if !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
```

Comment:
Why are you not removing the whole POSIX interface?

Reply:
Sorry, I meant functions other than write & read. Most of the POSIX interface should not be used for a minimal console. For example, you have removed the implementation for lseek, ftruncate, fsync etc. so why keep the declaration

Reply:
[Done](https://github.com/hugueskamba/mbed-os/compare/f3f1c0fdf36f8ca6cf191aa1c815dbeee0787f45..043ec8be3f169c45aa777b41356826e61b9565e5).